modules/adminui/plugins/formwidget/passwordeditor_adminlte/passwordeditor_adminlte.formwidget.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * @author    Raphaël MARTIN
+ * @copyright 2023 Raphaël MARTIN
+ *
+ * @link     https://jelix.org
+ * @licence  MIT
+ */
+require_once JELIX_LIB_PATH . 'plugins/formwidget/passwordeditor_html/passwordeditor_html.formwidget.php';
+
+class passwordeditor_adminlteFormWidget extends passwordeditor_htmlFormWidget {
+    use \Jelix\AdminUI\Form\WidgetTrait;
+
+    public function outputMetaContent($resp) {
+        if (!is_null($resp)) {
```

Comment:
why?  `$resp` is not supposed to be null. It if is null, then there is a bug somewhere

Reply:
I thought, when form is ajax loaded the response can be set null 

see https://github.com/jelix/jelix/blob/f8e7839756df9ca30a9483c1d75275e2ea8747fb/testapp/modules/testapp/controllers/sampleform.classic.php#L107

but the formbuilder handle this case
https://github.com/jelix/adminui-module/blob/deecf0437e85b9fc3d732d103bc64a91d4d95bd0/modules/adminui/plugins/formbuilder/adminlte/adminlte.formbuilder.php#L22

fixing